src/objectExplorer/objectExplorerService.ts

```diff
@@ -377,7 +377,8 @@ export class ObjectExplorerService {
             connectionCredentials = await connectionUI.showConnections(false);
         }
         if (connectionCredentials) {
-            if (ConnectionCredentials.isPasswordBasedCredential(connectionCredentials)) {
+            if (ConnectionCredentials.isPasswordBasedCredential(connectionCredentials) &&
```

Comment:
If the connection string doesn't have a password but is using SQL auth what does that experience look like? I would expect it to still query for the password at that point since we don't necessarily want to require users putting their password in the connection string.

Reply:
This assumes that if the connection is SQL Auth, it will have the credentials in the string. We just send the connection string as is to the sqltoolsservice just like ADS. Querying for the password in the connection string is something we can look into for future for both the extension and ADS.

Reply:
What's the experience like? I assume if they don't an error will pop up telling them that they need to enter a password or something similar?